packages/smalltalk/3.2.3/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+curl -L https://ftp.gnu.org/gnu/smalltalk/smalltalk-3.2.3.tar.gz -o smalltalk-3.2.3.tar.gz
+tar xzf smalltalk-3.2.3.tar.gz
+
+rm smalltalk-3.2.3.tar.gz
+
+cd smalltalk-3.2.3
+
+./configure && rm libc.la && make && make install && chmod +x gst && cd ..
```

Comment:
not a big fan of all of this on the same line, maybe just `bash -e`?

Reply:
o### _**k**_